packages/e2e/cypress.config.ts

```diff
@@ -24,7 +24,7 @@ export default defineConfig({
       // This is a chrome launch option which enables fake videos
       on('before:browser:launch', (_browser, launchOptions) => {
         launchOptions.args.push(
-          '--use-file-for-fake-video-capture=cypress/fixtures/faceRecording.mjpeg'
```

Comment:
Is there an associated _mjpeg_ file we should delete?

Reply:
We could delete it, but for some reason I felt it might be useful to keep it in case we ever need to switch to it. 
We luckily happened to have the y4m file on my desktop but would just be annoying to have to create it again. What do you think?